packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -627,6 +627,27 @@ export class DataStores implements IDisposable {
         }
     }
 
+    /**
+     * When running GC in tombstone mode, this is called to tombstone objects whose routes are unused. This enables
+     * testing scenarios with accessing deleted content without actually deleting content from summaries.
+     * @param unusedRoutes - The routes that are unused in all data stores in this Container.
+     */
+    public tombstoneUnusedRoutes(unusedRoutes: string[]) {
```

Comment:
As per my comment [here](https://github.com/microsoft/FluidFramework/pull/12259/files#r997455018), you could re-use deleteUnusedRoutes and add a param to it that tells whether to delete or tombstone. That way, you don't have to duplicate this code.

Reply:
Method removed